<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Colorpicker</class>
 <widget class="QMainWindow" name="Colorpicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>213</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>17</green>
        <blue>23</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>17</green>
        <blue>23</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>17</green>
        <blue>23</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>17</green>
        <blue>23</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>17</green>
        <blue>23</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>14</red>
        <green>17</green>
        <blue>23</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Color Picker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>49</red>
          <green>54</green>
          <blue>61</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>49</red>
          <green>54</green>
          <blue>61</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="63">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>154</y>
      <width>187</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>76</y>
      <width>61</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="rgblayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="rlabel">
       <property name="text">
        <string>R:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="r">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="glabel">
       <property name="text">
        <string>G:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="g">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="blabel">
       <property name="text">
        <string>B:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="b">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>76</y>
      <width>61</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="hsvlayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="hlabel">
       <property name="text">
        <string>H:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="h">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="s">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="v">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="slabel">
       <property name="text">
        <string>S:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="vlabel">
       <property name="text">
        <string>V:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="displayProfile">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>178</y>
      <width>194</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Display profile</string>
    </property>
   </widget>
   <widget class="QSlider" name="aperture">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>178</y>
      <width>127</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="displayInProfile">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>40</y>
      <width>210</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Native display values</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="mouseLocation">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>127</y>
      <width>126</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>14</red>
          <green>17</green>
          <blue>23</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>14</red>
          <green>17</green>
          <blue>23</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>(0,0)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="display_2">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>154</y>
      <width>104</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Aperture size</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>19</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Display profile</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="displayValues">
     <property name="title">
      <string>Display values</string>
     </property>
     <addaction name="as8bitValues"/>
     <addaction name="as10bitValues"/>
     <addaction name="separator"/>
     <addaction name="asFloatValues"/>
     <addaction name="asHexadecimalValues"/>
     <addaction name="asPercentageValues"/>
    </widget>
    <widget class="QMenu" name="magnify">
     <property name="title">
      <string>Magnification</string>
     </property>
     <addaction name="magnify1x"/>
     <addaction name="magnify2x"/>
     <addaction name="magnify3x"/>
     <addaction name="magnify4x"/>
     <addaction name="magnify5x"/>
    </widget>
    <addaction name="freeze"/>
    <addaction name="separator"/>
    <addaction name="displayValues"/>
    <addaction name="separator"/>
    <addaction name="magnify"/>
    <addaction name="separator"/>
    <addaction name="toggleMouseLocation"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="copyRGBAsText"/>
    <addaction name="copyHSVAsText"/>
    <addaction name="copyProfileAsText"/>
    <addaction name="separator"/>
    <addaction name="copyColorAsBitmap"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="openGithubReadme"/>
    <addaction name="openGithubIssues"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="viewMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="copyRGBAsText">
   <property name="text">
    <string>Copy RGB as text</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="copyHSVAsText">
   <property name="text">
    <string>Copy HSV as text</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="copyColorAsBitmap">
   <property name="text">
    <string>Copy Color as bitmap</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="freeze">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Freeze</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="toggleMouseLocation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show mouse location</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="copyProfileAsText">
   <property name="text">
    <string>Copy Display profile as text</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="as8bitValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as 8-bit</string>
   </property>
   <property name="toolTip">
    <string>Display as 8-bit</string>
   </property>
  </action>
  <action name="as10bitValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as 10-bit</string>
   </property>
   <property name="toolTip">
    <string>Display as 10-bit</string>
   </property>
  </action>
  <action name="asFloatValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as Float</string>
   </property>
   <property name="toolTip">
    <string>Display as Decimal</string>
   </property>
  </action>
  <action name="asHexadecimalValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as Hexadecimal</string>
   </property>
   <property name="toolTip">
    <string>Display as Hexadecimal</string>
   </property>
  </action>
  <action name="asPercentageValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>as Percentage</string>
   </property>
   <property name="toolTip">
    <string>Display as Percentage</string>
   </property>
  </action>
  <action name="magnify1x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 100%</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="magnify2x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 200%</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="magnify3x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 400%</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="magnify4x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 800%</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="magnify5x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5x</string>
   </property>
   <property name="toolTip">
    <string>Zoom 1600%</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="openGithubReadme">
   <property name="text">
    <string>Open Github README</string>
   </property>
   <property name="toolTip">
    <string>Open Github README</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="openGithubIssues">
   <property name="text">
    <string>Open Github Issues</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
